tiledb/sm/filesystem/s3.h

```diff
@@ -505,6 +529,47 @@ class S3 {
     int upload_part_num;
   };
 
+  /**
+   * This type holds the location and the size of an intermediate chunk on S3
+   * These chunks are used with remote global writes according to the
+   * algorithm below:
+   *
+   * Intermediate chunks are persisted across cloud executors via serialization
+   * When a new write arrives via global_order_write(URI):
+   * - look if there is a multipart state for the URI, if not,
+   *   create one and initiate multipart request
+   * - look at the sizes of all intermediate chunks persisted before
+   *   for this URI
+   * - If sum(intermediate_chunks[i].size) + current_chunk_size < 5MB
```

Comment:
Change "sum(intermediate_chunks[i].size) + current_chunk_size" to human language like, if the sum of all intermediate chunks and the current chunk is less than 5MB.

Reply:
"- persist the new chunk as an intermediate..." To follow the convention below, this should be a) persist...

Reply:
NIT: I'd like to see all bullet points start with a capital letter and end with a `.`. This would help me to read this comment faster.